src/utils/isNextMonth.js

```diff
@@ -0,0 +1,6 @@
+import moment from 'moment';
+
+export default function isNextMonth(a, b) {
+  if (!moment.isMoment(a) || !moment.isMoment(b)) return false;
+  return b.isSame(a.clone().add(1, 'month'), 'month');
```

Comment:
I've found `isSame` to be slow even when it's got a 'month' or 'day' or whatever argument. Maybe let's do:
```
return b.month() === a.clone().add(1, 'month').month();
```

Reply:
Only issue there is that a = 2017-01-01, b = 2018-02-01 would return true when it would be false. May not be a thing based on how the function is being called, but could be an issue down the line. To me, `isSame` may be slower, but it is safer.

Perhaps, if performance is a concern, checking a.year === b.year and a.month === b.month might help?

Reply:
good call!